### DISCUSSION TITLE
Send the amount of servings when doing an action

### DISCUSSION THREAD

**Discussion by felixschndr** [mealie-recipes/mealie#4737] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

Mealie supports [recipe actions](https://docs.mealie.io/documentation/getting-started/features/#recipe-actions). I use them to build a bring integration that does not require the Mealie instance to be [publicly accessible](https://github.com/felixschndr/mealie-bring-api). [The users of it (and myself](https://github.com/felixschndr/mealie-bring-api/issues/15) would like that the amounts of foods should be changed accordingly to the number of servings specified in the UI, however this data is not sent in the `POST` request.

When the recipe actions were implemented I had a discussion with the one who introduced them but I can't find the conversation any more.

### Please provide a concise description of the feature that would resolve your issue.

The amount of servings presented in the UI should be sent within in the `POST` request, either in the amounts themselves or as a separate field.
E.g.
* recipe contains 100 ml milk and amount of servings is set to 1: 100 ml milk or `amount_of_servings`=1
* recipe contains 100 ml milk and amount of servings is set to 2: 200 ml milk or 100 ml milk and `amount_of_servings`=2

Since the client sends out the `POST` request and not the server *I guess* it should not be that difficult to get the current chosen servings amount and send it as a separate field.

### Please consider and list out some caveats or tradeoffs made in your design decision

I guess there can't be any

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [X] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by michael-genson** [comment]
Duplicate of https://github.com/mealie-recipes/mealie/discussions/3610
Now that https://github.com/mealie-recipes/mealie/pull/4489 is merged this should be trivial!
